config/vufind/KohaRest.ini

```diff
@@ -69,7 +69,8 @@ HMACKeys = item_id:holdtype:level
 defaultRequiredDate = 0:0:2
 
 ; extraHoldFields - A colon-separated list used to display extra visible fields in the
-; place holds form. Supported values are "requiredByDate" and "pickUpLocation"
+; place holds form. Supported values are "startDate", "requiredByDate" and
+; "pickUpLocation"
```

Comment:
That'd make sense. We'd still have to list the supported values in each .ini file as they differ per driver, but further explanation could be on a wiki page. Maybe a sub-page of https://vufind.org/wiki/configuration:ils ?

Reply:
Yes, that makes sense to me. Then we could link to this page from the driver spec as well as from the .ini files, and it would define a common vocabulary of settings that the individual drivers could choose from as appropriate. Let me know if you need me to create a stub page for you.

Reply:
Added page https://vufind.org/wiki/configuration:ils:holds

Reply:
Thanks, this is great! The only problem was that the extraHoldFields text was not displaying neatly, but I reformatted it as a table and now it looks fine.

Reply:
Awesome, the table is much nicer!

Reply:
...so now the last step is to link to this page from the relevant .ini files. Let me know if you'd like help getting that done!

Reply:
I can add the links, but if you'd like to update the comments in the ini files as well, I'd appreciate help.

Reply:
Why don't you add it to KohaRest.ini and Demo.ini as part of this pull request, since you're already modifying those two files, and I'll add it to all the other ones that aren't part of the PR?

Reply:
...actually, I'll take care of KohaRest.ini too, since I'll just add a comment above the `[Holds]` section which won't conflict with anything else. I can also paste the same comment into Demo.ini here. I'll let you know when it's all done!

Reply:
Thanks!

Reply:
Okay, all done -- since it's a trivial (comments-only) change, I went ahead and committed directly here: https://github.com/vufind-org/vufind/commit/1751f32411eeb6265eded161bdd31c7a581c85e4